extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,127 @@
+| keywords          | [string]                                           | Unordered list of one or more commonly used or formalized word(s) or phrase(s) used to describe this dataset                 |
+| identifier        | string                                             | **REQUIRED.** Tile matrix set identifier. This can be used as the `{TileMatrixSet}` template parameter.                      |
+| boundingBox       | [BoundingBox Object](#bounding-box-object)         | Minimum bounding rectangle surrounding the tile matrix set, in the supported CRS                                             |
+| supportedCRS      | string                                             | **REQUIRED.** Reference to one coordinate reference system (CRS)                                                             |
+| wellKnownScaleSet | string                                             | Reference to a well-known scale set.                                                                                         |
+| tileMatrix        | [[Tile Matrix Object](#tile-matrix-object)]        | **REQUIRED.** An array of tile matrix objects, describing the tile matrices of the resolution layers of the tile matrix set. |
+| pixelBuffer       | [Pixel Buffer Object](#pixel-buffer-object)        | An optional default pixel buffer description object. By default, no pixel buffers are used.                                  |
+
+### Bounding Box Object
+
+| Field Name  | Type     | Description                                                        |
+| ----------- | -------- | ------------------------------------------------------------------ |
+| type        | string   | **REQUIRED.** Type of the Bounding Box. Fixed to `BoundingBoxType` |
+| lowerCorner | [number] | **REQUIRED.** The lower corner of this bounding box.               |
+| upperCorner | [number] | **REQUIRED.** The upper corner of this bounding box.               |
+
+### Tile Matrix Object
+
+The description of a single tile matrix (zoom level) of the tile pyramid.
+
+| Field Name          | Type                                                          | Description                                                                                                                                                       |
+| ------------------- | ------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type                | string                                                        | **REQUIRED.** Type of the Tile Matrix. Fixed to `TileMatrixType`                                                                                                  |
```

Comment:
A field which only allows a single value seems note very useful and could probably be omitted.

Reply:
Agreed. This was removed with the re-treading of the 2D TMS spec.